controller_manager:
    ros__parameters:
        update_rate: 100
        use_sim_time: true
        # 注册状态发布器
        joint_state_broadcaster:
            type: joint_state_broadcaster/JointStateBroadcaster
            use_sim_time: true
        # 注册速度控制器
        force1_velocity_controller:
            type: velocity_controllers/JointVelocityController

# 配置速度控制器参数
force1_velocity_controller:
    ros__parameters:
        type: velocity_controllers/JointVelocityController
        joint: force1
        pid:
            p: 100.0
            i: 0.01
            d: 10.0